master/buildbot/worker/base.py

```diff
@@ -17,7 +17,6 @@
 from email.message import Message
 from email.utils import formatdate
 
-from future.utils import itervalues
```

Comment:
its made automatically by isort.

For some reason isort does not run on travis as it should, so we start taking technical debt about it.

That said. I would expect that future is part of the first. So maybe it is the config of my editor which cannot find the isort configuration